app/components/public/session-item.hbs

```diff
@@ -48,7 +48,7 @@
           {{/if}}
         </div>
       {{else}}
-        <div class="left floated eleven wide column">
+        <div class="left floated eleven wide column" style="padding-bottom: 0 !important;" >
```

Comment:
Use class

Reply:
> scss classes are overrided (or not working) due to ` ui grid wide column` that's why using instyling

I tried in utils.scss but it's not working @iamareebjamal

Reply:
It'll work